internal/dcache/rpc/server/handler.go

```diff
@@ -1822,7 +1854,7 @@ func (h *ChunkServiceHandler) StartSync(ctx context.Context, req *models.StartSy
 	//
 	mvInfo := rvInfo.getMVInfo(req.MV)
 	if mvInfo == nil {
-		errStr := fmt.Sprintf("%s/%s not hosted by this node", rvInfo.rvName, req.MV)
+		errStr := fmt.Sprintf("%s is not part of %s", rvInfo.rvName, req.MV)
```

Comment:
not hosted by is more appropriate, since user is targeting startsync for an mv replica

Reply:
Updated